amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -0,0 +1,328 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at:
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the
+# License.
+"""A repeatable benchmark tool for ion-python implementation.
+
+Usage:
+    ion_python_benchmark_cli.py write [--api <api>] [--iterator <bool>] [--warmups <int>] [--iterations <int>]
```

Comment:
The `--iterator` option is only applicable to the `read` command, right?

Reply:
Thanks for catching up this!